UnitsNet/UnitDefinitions/ElectricCurrentDensity.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "ElectricCurrentDensity",
+  "BaseUnit": "AmperePerMeterSecondSquared",
+  "XmlDoc": "In electromagnetism, current density is the electric current per unit area of cross section.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Current_density",
+  "Units": [
+    {
+      "SingularName": "AmperePerMeterSecondSquared",
```

Comment:
Please use `SquareMeter` instead of instead of `MeterSecondSquare` to be consistent with other units

Reply:
AmperePerMeterSecondSquared was clearly a wrong unit (a typo probably).
However you're true about the consistency issue and we will modify as requested.